### ISSUE TITLE
DSMR 5, double gas meter not working

### ISSUE THREAD

**Issue by MariekeVDA** [dupondje/homeassistant-dsmr#9]
Hello,
My active gas meter is connected to mbus2, however I do not get it to work.
When I set my DSMR version to 5 I see a constant value of 0,11m3, which corresponds to the first mbus value.
When I set my DSMR version to 5B it creates 2 gas meter entities: mbus1 and mbus2. However both show unknown.
Do I understand it correctly that multiple mbussus is only supported for DSMR version 5B?

See below for a telegram, the second value of 06990.193*m3 is my actual gasmeter. 

```

2023-10-23 20:46:32.471 DEBUG (MainThread) [dsmr_parser.clients.protocol] got telegram: /XMX5LGBBLB2412107893

1-3:0.2.8(50)
0-0:1.0.0(231023204430S)
0-0:96.1.1(xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx)
1-0:1.8.1(006550.417*kWh)
1-0:1.8.2(006605.954*kWh)
1-0:2.8.1(002074.734*kWh)
1-0:2.8.2(004419.186*kWh)
0-0:96.14.0(0002)
1-0:1.7.0(00.261*kW)
1-0:2.7.0(00.000*kW)
0-0:96.7.21(00003)
0-0:96.7.9(00006)
1-0:99.97.0(6)(0-0:96.7.19)(171009153324S)(0003038715*s)(180107022907W)(0000010862*s)(180218190425W)(0000005831*s)(180420131843S)(0000010329*s)(200401064737S)(0000002692*s)(230627022747S)(0000000214*s)
1-0:32.32.0(00848)
1-0:32.36.0(00001)
0-0:96.13.0()
1-0:32.7.0(225.0*V)
1-0:31.7.0(001*A)
1-0:21.7.0(00.261*kW)
1-0:22.7.0(00.000*kW)
0-1:24.1.0(003)
0-1:96.1.0(xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx)
0-1:24.2.1(170904112508S)(00000.110*m3)
0-2:24.1.0(003)
0-2:96.1.0(xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx)
0-2:24.2.1(231023204000S)(06990.193*m3)
!69FF

```

**Comment by dupondje** [comment]
Hi,

This is a bug indeed. We are having good progress on getting the changes into HA Core itself.
The thing is that gas values are on x-x:24.2.3 in BE and on x-x:23.2.1 in NL. That's why the values show None.

This needs to be handled in the code.

**Comment by dupondje** [comment]
Hi,

This is a bug indeed. We are having good progress on getting the changes into HA Core itself.
The thing is that gas values are on x-x:24.2.3 in BE and on x-x:23.2.1 in NL. That's why the values show None.

This needs to be handled in the code.

**Comment by MariekeVDA** [comment]
Ah good to know that this is not something on my end. 
thank you for your work on getting this into HA Core!

**Comment by MariekeVDA** [comment]
Do you advice me to create another issue at home assistant core?

**Comment by dupondje** [comment]
No not needed atm.
Once https://github.com/ndokter/dsmr_parser/pull/142 is merged and a new release is out, then dsmr integration in HA can be adapted to it and it should work :) Just waiting for things to get merged now.

**Comment by screenagerbe** [comment]
will the water meter also be added with this new release ?

Because on my test system it appears it doesn't with the default integration 🙁
As I don't have the gas meter installed yet, I assume I can still use your v1.0.5

**Comment by MariekeVDA** [comment]
Hi @dupondje,

I see that your work is merged in HA, congrats!
Do you have any update on this issue?
